pkg/storkctl/clusterdomainsstatus.go

```diff
@@ -0,0 +1,82 @@
+package storkctl
+
+import (
+	"fmt"
+	"io"
+
+	storkv1 "github.com/libopenstorage/stork/pkg/apis/stork/v1alpha1"
+	"github.com/portworx/sched-ops/k8s"
+	"github.com/spf13/cobra"
+	"k8s.io/kubernetes/pkg/kubectl/cmd/util"
+	"k8s.io/kubernetes/pkg/kubectl/genericclioptions"
+	"k8s.io/kubernetes/pkg/printers"
+)
+
+var clusterDomainsStatusColumns = []string{"NAME", "ACTIVE", "INACTIVE", "CREATED"}
+var clusterDomainsStatusSubcommand = "clusterdomainsstatus"
+var clusterDomainsStatusAliases = []string{"cds"}
+
+func newGetClusterDomainsStatusCommand(cmdFactory Factory, ioStreams genericclioptions.IOStreams) *cobra.Command {
+	getClusterDomainsStatusCommand := &cobra.Command{
+		Use:     clusterDomainsStatusSubcommand,
+		Aliases: clusterDomainsStatusAliases,
+		Short:   "Get cluster domain statuses",
+		Run: func(c *cobra.Command, args []string) {
+			cdStatuses := new(storkv1.ClusterDomainsStatusList)
+			var err error
+			if len(args) > 0 {
+				for _, clusterID := range args {
+					cds, err := k8s.Instance().GetClusterDomainsStatus(clusterID, "kube-system")
```

Comment:
Don't need to pass in namespace since the object is cluster scoped

Reply:
I don't see the `+gencclient:nonNonspaced` on the `ClusterDomainsStatus` CRD. If this is cluster scope, the CRD and the generated code needs change.